JSDemos/Demos/TabPanel/Overview/description.md

```diff
@@ -1,5 +1,13 @@
-<<<<<<< HEAD
-The TabPanel is&nbsp;a&nbsp;UI component consisting of&nbsp;the Tabs and the MultiView UI components. It&nbsp;automatically synchronizes the selected tab with the currently displayed view and vice versa. In&nbsp;this demo, the appearance of&nbsp;views and tabs is&nbsp;defined by&nbsp;custom templates.
-=======
-The TabPanel is&nbsp;a&nbsp;UI component consisting of&nbsp;the Tabs and the MultiView components. It&nbsp;automatically synchronizes the selected tab with the currently displayed view and vice versa. In&nbsp;this demo, the appearance of&nbsp;views and tabs is&nbsp;defined by&nbsp;custom templates.
->>>>>>> e74b8529 (Cut off "UI" from "UI component" is certain cases (#612))
+The [TabPanel](/Documentation/ApiReference/UI_Components/dxTabPanel/) UI component consists of the [Tabs](/Documentation/ApiReference/UI_Components/dxTabs/) and [MultiView](/Documentation/ApiReference/UI_Components/dxMultiView/) components. TabPanel automatically synchronizes the selected tab with the currently displayed view and vice versa. This demo demonstrates the following TabPanel features:      
+
+### Bind the TabPanel to Data       
+
+The [dataSource](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/#dataSource) property binds the TabPanel to data. To specify a custom template for all the TabPanel views, use the [itemTemplate](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/#itemTemplate). To do the same for the tabs, use [itemTitleTemplate](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/#itemTitleTemplate).
```

Comment:
1)Why use custom templates? The data objects use custom field names. This explanation would come in handy. The [Customize Item Appearance](https://js.devexpress.com/Documentation/Guide/UI_Components/TabPanel/Customize_Item_Appearance/) is a good example.
2) What are the standard field names? As far as I could understand, they are listed in the items[] prop. description, but not the dataSource description (should dataSource be extended?). 
3) Should items[] be mentioned?

Reply:
Sorry, I did not understand your questions. 

1. Do you suggest to add something like 'You can define custom templates for data in TabPanel'? And what custom names use the data objects?
2. Which standard fields do you mean? Is it about the scheduler internal things that you described in the Resources demo? items and dataSource are alternatives to each other and developers cannot use them simultaneously.
3. No, see the previous point.

Reply:
I guess that you confuse the items-dataSource pair in collection components(List, Toolbar, tabPanel) is similar to the columns-dataSource in grids. In the latter case, dataSource provides data and columns[] specifies which data to use. 

But the collection components( work in a different way. You can specify data/layout only once, and to customize this data you need to use templates or specify markup directly using items[].